modules/analysis/rosbag-image-pipeline-sfn/stack.py

```diff
@@ -38,7 +38,7 @@ def __init__(
         lane_detection_config: Dict[str, str],
         emr_job_config: Dict[str, str],
         stack_description: str,
-        image_topics: List[str],
+        image_topics: Any,
```

Comment:
Probably don't want Any....right?

Reply:
hmm json.loads() yields `Any` so may be a mypy issue